src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -223,6 +227,37 @@ public boolean postString(final Locale locale, final String component,
 		return cs.getComponentTranslation();
 	}
 
+
+    /**
+     * get multiple component's translations from VIP server
+     *
+     * @param locales
+     *            language tags to get the translations of them
+     * @param components
+     *            names of the components
+     * @return a map contains all translations of the components of specified locales
```

Comment:
give more details to the two maps about what's the key and value.

Reply:
done